src/network.cpp

```diff
@@ -150,18 +152,18 @@ void IRAM_ATTR RemoteLoopEntry(void *)
 // ConnectToWiFi
 //
 // Connect to the pre-configured WiFi network.  
-//
-// BUGBUG I'm guessing this is exposed in all builds so anyone can call it and it just returns false if wifi
-// isn't being used, but do we need that?  If no one really needs to call it put the whole thing in the ifdef
 
+#if ENABLE_WIFI

```

Comment:
As before, I think it's clearer if the code between this `#if ENABLE_WIFI` directive and its corresponding `#endif` is indented.